packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -861,7 +861,10 @@ class ScrollableState extends State<Scrollable>
 
   void _handleDragDown(DragDownDetails details) {
     assert(_drag == null);
-    assert(_hold == null);
+    // _hold might be non-null if the scroll position is currently animating.
+    if (_hold != null) {
```

Comment:
This should be null. Do we have a scenario where it crashes here?

Reply:
@victorsanni , I am also curious what the sequence of event that causes this to be none null. My suspicion is that the previous event was cancelled before starting a new one, but would be good to have confirmation.